README.md

```diff
@@ -19,6 +19,12 @@ A data build tool
 
 # configure dbt file paths (relative to dbt_project.yml)
 
+# the package config is _required_. If other packages import this package, 
+# the name given below is used to reference this package
+package:
```

Comment:
I don't think the package namespace is necessary here.  The name and version are properties of the project, lets make them top-level objects